src/github/nag.rs

```diff
@@ -898,6 +1003,14 @@ impl<'a> RfcBotComment<'a> {
         }
     }
 
+    fn couldnt_add_label<'b>(msg: &mut String, author: &'b GitHubUser, label: &str) {
+        msg.push_str("\n\n*psst @");
```

Comment:
I'd be ok with this being a call to `format!` -- in hindsight I think it was a big mistake to be concerned about allocation performance here.

Reply:
Hmm... can we fix this later as a more general refactoring of message formats and potential templating?